### PR TITLE
OSDOCS-477 updates to Azuremachinesets

### PR DESCRIPTION
**Pull request by kalexand-rh**
Follow up to https://github.com/openshift/openshift-docs/pull/16805

@sunzhaohua2, will you PTAL?

### CODE REVIEW

File: modules/machineset-yaml-azure.adoc [hunk 1]
```diff
@@ -7,8 +7,8 @@
 =  Sample YAML for a MachineSet Custom Resource on Azure
 
 This sample YAML defines a MachineSet that runs in the `centralus1`
-Microsoft Azure region and creates nodes that are labeled with
-`node-role.kubernetes.io/<role>: ""`
+Microsoft Azure zone in the `centralus` regions and creates nodes that are
```
Comment (sunzhaohua2) [inline modules/machineset-yaml-azure.adoc]: runs in the `1` Microsoft Azure zone in the `centralus` region

**Comment by kalexand-rh** [comment]
Thank you @sunzhaohua2! Please let me know if you see anything else to change.

**Comment by sunzhaohua2** [comment]
LGTM, thank you.

**Pull request merged**